client/src/js/services/SubsidiesService.js

```diff
@@ -0,0 +1,87 @@
+angular.module('bhima.services')
+  .service('SubsidiesService', SubsidiesService);
+
+SubsidiesService.$inject = ['$http', 'util'];
+
+function SubsidiesService($http, util) {
+  var service = this;
+  var baseUrl = '/subsidies/';
+
+  service.read = read;
+  service.create = create;
+  service.update = update;
+  service.delete = del;
+
+  /**
+  * @desc Get an id (optionnal) and return back a list of subsidies or an subsidy
```

Comment:
This are really great comments!

One small thing: JavaScript doesn't have a type or object called `Integer`.  Instead of writing `@param {Integer}` it would be better to write `@param {Number}`, since that is JavaScript's numerical type.

Otherwise, this is excellent code documentation!